frontend/src/scenes/experiments/RunningTimeCalculator/RunningTimeCalculatorModal.tsx

```diff
@@ -113,12 +114,22 @@ export function RunningTimeCalculatorModal(): JSX.Element {
                                             Last {TIMEFRAME_HISTORICAL_DATA_DAYS} days
                                         </div>
                                     </div>
-                                    <div>
-                                        <div className="card-secondary">Avg. events per user</div>
-                                        <div className="font-semibold">
-                                            ~{humanFriendlyNumber(averageEventsPerUser || 0, 0)}
+                                    {averageEventsPerUser !== null && (
+                                        <div>
+                                            <div className="card-secondary">Avg. events per user</div>
+                                            <div className="font-semibold">
+                                                ~{humanFriendlyNumber(averageEventsPerUser || 0, 0)}
+                                            </div>
                                         </div>
-                                    </div>
+                                    )}
+                                    {averagePropertyValuePerUser !== null && (
+                                        <div>
+                                            <div className="card-secondary">Avg. property value per user</div>
+                                            <div className="font-semibold">
+                                                ~{humanFriendlyNumber(averagePropertyValuePerUser, 0)}
+                                            </div>
+                                        </div>
+                                    )}
                                     <div>
                                         <div className="card-secondary">Estimated variance</div>
                                         <div className="font-semibold">~{humanFriendlyNumber(variance, 0)}</div>
```

Comment:
I think maybe it's more meaningful to show standard deviation to the user, as it has a much more intuitive meaning